guides/common/modules/proc_enabling-the-template-sync-plugin.adoc

```diff
@@ -2,12 +2,23 @@
 = Enabling the Template Sync plugin
 
 .Procedure
-. To enable the plugin on your {ProjectServer}, enter the following command:
+. Enable the plugin on your {ProjectServer}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-installer} --enable-foreman-plugin-templates
 ----
-. To verify that the plugin is installed correctly, ensure *Administer* > *Settings* includes the *Template Sync* menu.
-. Optional: In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Template Sync* to configure the plugin.
+. Optional: Enable the Hammer CLI plugin on your {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --enable-foreman-cli-templates
```

Comment:
For satellite we have `satellite-cli` which already pulls the package in which means this section should be hidden for satellite.